### PR TITLE
List users layout

### PR DESCRIPTION
**Pull request by mvn-tamnguyen4-dn**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-10
- Task ID: #10.1

## Migrations
 NO

## Impacted Areas in Application
- resource/lang/en/dashboard.php

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/backend/users/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+              <tbody>
+              <tr>
+                <td>1</td>
+                <td>AT-00001</td>
+                <td>Employee 1</td>
+                <td>a@asiantech.vn</td>
+                <td>3</td>
+                <td>5</td>
+              </tr>
+              </tfoot>
```
Comment (at-minhphan) [inline resources/views/backend/users/index.blade.php]: sửa lại </tbody> chứ không phải </tfoot>

File: resources/views/backend/users/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
+@extends('backend.layouts.main')
+
+@section('title','Users')
```
Comment (at-xinhnguyen) [inline resources/views/backend/users/index.blade.php]: please use `@section('title', trans('dashboard.users'))` instead of this

**Comment by at-xinhnguyen** [comment]
LGTM

File: resources/lang/en/dashboard.php [hunk 3]
```diff
@@ -6,4 +6,12 @@
...
     'main_navigation' => 'MAIN NAVIGATION',
     'online'=> 'Online',
+    'list_users' => 'List Users',
+    'users_table' => 'Users List Table',
+    'employee_code' => 'Employee code',
+    'employee_name' => 'Name',
+    'employee_email' => 'Email',
+    'total_donated' => 'Total donated books',
+    'total_borrowed' => 'Total borrowed books',
+    'user_title' => 'Users'
```
Comment (at-xinhnguyen) [inline resources/lang/en/dashboard.php]: mấy cái này nên chuyển qua file en/user

**Pull request merged**